mllib/src/main/scala/org/apache/spark/mllib/clustering/LDA.scala

```diff
@@ -269,6 +270,8 @@ class LDA private (
    */
   @Since("1.3.0")
   def setCheckpointInterval(checkpointInterval: Int): this.type = {
+    require(checkpointInterval == -1 || checkpointInterval > 0,
```

Comment:
The is no document direct for -1.
I add -1 according to @yanboliang 's comments.